pymongo/client_session.py

```diff
@@ -381,7 +383,8 @@ def with_transaction(self, callback, read_concern=None, write_concern=None,
                          read_preference=None):
         """Execute a callback in a transaction.
 
-        .. code-block:: python
+        This method starts a transaction on this session, executes ``callback``
+        once, and then commits the transaction. For example::
 
```

Comment:
Why remove the `.. code-block:: python` statement? I'm guessing you had that in there so the docs rendered with syntax-highlighting? Is there a reason we don't want that to happen now?

Reply:
A trailing `::` works the same and is shorter.